### PR TITLE
seminars: add literature records

### PR DESCRIPTION
**Pull request by monaawi**
* INSPIR-3571

### CODE REVIEW

File: backend/inspirehep/submissions/marshmallow/seminars.py [hunk 1]
```diff
@@ -82,6 +89,13 @@ def convert_to_form_data(self, data):
         end_datetime = data.get("end_datetime")
         form_end_datetime = iso_utc_to_local_form_datetime(end_datetime, timezone)
 
+        literature_records = missing
+        if "literature_records" in data:
+            literature_records = [
+                PidStoreBase.get_pid_from_record_uri(rec["record"]["$ref"])[1]
+                for rec in data.get("literature_records")
```
Comment (drjova) [inline backend/inspirehep/submissions/marshmallow/seminars.py:96]:
```suggestion
                for rec in data.get("literature_records", [])
```
and you could drop the if

File: backend/inspirehep/submissions/marshmallow/seminars.py [hunk 2]
```diff
@@ -97,6 +111,7 @@ def convert_to_form_data(self, data):
             "timezone": timezone,
             "abstract": data.get_value("abstract.value", missing),
             "keywords": data.get_value("keywords.value", missing),
+            "literature_records": literature_records,
```
Comment (drjova) [inline backend/inspirehep/submissions/marshmallow/seminars.py:114]:
```suggestion
            "literature_records": literature_records or missing
```

File: ui/src/seminars/components/LiteratureRecordsList.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    <Link
+      data-test-id="author-link"
+      to={`${LITERATURE}/${literatureRecord.get('control_number')}`}
+    >
+      <LiteratureTitle title={title} />
+    </Link>
+  );
+}
+
+function LiteratureRecordsList({ literatureRecords }) {
```
Comment (drjova) [inline ui/src/seminars/components/LiteratureRecordsList.jsx:24]: don't we have a similar component? 🤔

File: backend/inspirehep/submissions/errors.py [hunk 4]
```diff
@@ -20,7 +20,16 @@ class WorkflowStartError(RESTException):
...
+        self.description = description
+
     code = 400
-    description = (
-        "Something went wrong while proccessing your data, please try again later."
-    )
+
+
+class InvalidDataError(Exception):
+    pass
```
Comment (drjova) [inline backend/inspirehep/submissions/errors.py:35]: why we want pass here?

File: ui/src/seminars/components/LiteratureRecordsList.jsx (hunk 3 above)
Comment (monaawi) [reply ui/src/seminars/components/LiteratureRecordsList.jsx:24]: I checked and we don't seem to have something like this

File: backend/inspirehep/submissions/errors.py (hunk 4 above)
Comment (monaawi) [reply backend/inspirehep/submissions/errors.py:35]: I didn't need to add anything more. What should I add?

Comment (drjova) [reply backend/inspirehep/submissions/errors.py:35]: Sorry I thought it was `RESTException`

**Pull request merged**